src/magic.cpp

```diff
@@ -243,6 +261,13 @@ void spell_type::check_consistency()
             debugmsg( string_format( "ERROR: %s has positive increment and base_casting_time > final_casting_time",
                                      sp_t.id.c_str() ) );
         }
+        if( !sp_t.additional_spells.empty() ) {
```

Comment:
You don't need this check, just iterate over the container. If it's empty, nothing will be done. This check is just redundant.